Update smtp-email-setup.rst

svelle opened the pull request:
Re: https://github.com/mattermost/docs/issues/3303

justinegeffen commented on source/install/smtp-email-setup.rst:
```diff
@@ -163,25 +163,8 @@ Checking your SMTP server is reachable
 -  You must run the following commands from the same machine or virtual
```
It looks like the "following" commands have been removed from this section. Are there still commands that can/must be run? If not, perhaps something like this could work: 

- Ensure you are running commands from the same machine or virtual instance where mattermost/bin/mattermost is located. 
- Attempt to telnet to the email service to make sure the server is reachable.
- Ping the smtp server to see if a connection can be established.

svelle replied:
So the problem here, and why I made the change, is that we don't install telnet inside the docker containers so we can't advise using that to verify the connection. 

I think referring to ping is probably the best thing we can do at this point apart from telling people to install telnet themselves and test it that way.

justinegeffen commented:
Hi @svelle. Following up on this to confirm whether you've had a chance to look at the feedback. :)

svelle commented:
Hi @justinegeffen still struggling with github notifications so sorry if I missed it, I'll check it out now and respond to you within the next few minutes ;)

justinegeffen commented:
No problem at all - I've also had some issues hence the follow up. :)

svelle commented:
@justinegeffen I completely redid the section and removed the direct reference to docker commands. 
Instead I added a note about telnet not being installed in the docker images and added some advice on how to work from there.

Let me know what you think :)

justinegeffen commented on source/install/smtp-email-setup.rst:
```diff
@@ -182,6 +180,10 @@ Checking your SMTP server is reachable
+       
+
+.. note:: 
+  We're not installing telnet by default on the official docker images, so you either need to use ``ping`` on those or install telnet yourself either directly in the machine or by modifying the dockerfile.
```
```suggestion
  As we're not installing telnet by default on the official docker images you either need to use ``ping`` on those or install telnet yourself either directly modifying the dockerfile.
```

Not sure what machine we're referring to so I removed it - happy to put it back if we can clarify.
